test/execution_test.ts

```diff
@@ -565,6 +571,28 @@ describe('Execution', () => {
           );
         });
 
+        it('sync with incremental chaining works', async () => {
+          await executeFormulaOrSyncFromCLI({
+            vm,
+            formulaName: 'Students>incremental',
+            params: ['Mr. Incremental'],
+            manifest: fakePack,
+            manifestPath: '',
+            bundleSourceMapPath,
+            bundlePath,
+            contextOptions: {useRealFetcher: false},
+          });
+          const result = mockPrintFull.args[0][0];
+
+          // What happens if we have multiple incremental syncs?
+          // Assuming we sync the first page in the normal sync, and then passing
+          // >incremental just gets the next page.
```

Comment:
It's a good question. I think I gravitate towards running a "complete" incremental sync, that is, kick off a sync loop that has a `previousCompletion` provided in the context, and run it like normal, in a loop until there is no more `continuation`. Print all of the results from the incremental loop.

Reply:
Yep, good summary, we're on the same page.